luatest/hooks.lua

```diff
@@ -172,7 +175,9 @@ function export.patch_runner(Runner)
             run_named_test_hooks(self, test, 'before_test')
 
             if test:is('success') then
+                log.verbose('Start %s test', test.name)
```

Comment:
1. I would convert it to `info`.
2. I propose to use the following format:

`` Start test `%s` ``

Reply:
I've fixed that.